samples/snippets/subscriber.py

```diff
@@ -160,6 +160,28 @@ def create_push_subscription(project_id, topic_id, subscription_id, endpoint):
     # [END pubsub_create_push_subscription]
 
 
+def create_subscription_with_ordering(project_id, topic_id, subscription_id):
+    """Create a subscription with dead letter policy."""
+    # [START pubsub_enable_subscription_ordering]
+    from google.cloud import pubsub_v1
+
+    # TODO(developer): Choose an existing topic.
+    # project_id = "your-project-id"
+    # topic_id = "your-topic-id"
+    # subscription_id = "your-subscription-id"
+
+    subscriber = pubsub_v1.SubscriberClient()
+    topic_path = subscriber.topic_path(project_id, topic_id)
```

Comment:
`topic_path()` does not exist anymore on subscriber client, had to fix quite a few of these in #158. :)

Reply:
That is a massive PR! Thank you for the heads-up! I will aim to merge this one after that one and the changes are in.